plugins/alignments/src/PileupRenderer/PileupRenderer.ts

```diff
@@ -309,9 +309,12 @@ export default class PileupRenderer extends BoxRendererType {
             }
           } else if (
             mismatch.type === 'hardclip' ||
-            (!showSoftClip && mismatch.type === 'softclip')
+            mismatch.type === 'softclip'
           ) {
             ctx.fillStyle = mismatch.type === 'hardclip' ? 'red' : 'blue'
+            if (!seq) {
+              ctx.fillStyle = 'black'
```

Comment:
What about using `theme.palette.text.primary`? It'll still show up as black, but might help out if we decide to enable dark mode in the future.

Reply:
If you think that makes sense I can do this. The alternative is creating a whole new palette of colors just for the clip indicators but this is probably fine

Reply:
Added on branch